### PR TITLE
Soc 2669

### PR DESCRIPTION
**Pull request by justnpT**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsAnon.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import com.wikia.webdriver.common.core.annotations.RelatedIssue;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsAnon.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/89f8a059ddffd62c6b41bf87279e9ef3b9e6533e/src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java#L6) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsAnon.java (hunk 1 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsAnon.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditAccountTests.java [hunk 2]
```diff
@@ -29,15 +32,17 @@ public void EditAccount_001_closeAccount() {
...
   }
 
-  @Test(groups = "EditAccountTest", dependsOnMethods = "EditAccount_001_closeAccount")
+  @Test(dependsOnMethods = "EditAccount_001_closeAccount")
   public void EditAccount_002_verifyAccountClosed() {
     WikiBasePageObject base = new WikiBasePageObject();
-    SpecialUserLoginPageObject login = base.openSpecialUserLoginOnWiki(wikiURL);
-    login.login(credentials.userNameClosedAccount, credentials.passwordClosedAccount);
-    login.verifyClosedAccountMessage();
+    SpecialUserLoginPageObject login = new SpecialUserLoginPageObject(driver);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditAccountTests.java]: MAJOR Remove this unused "login" local variable. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)

File: src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java [hunk 3]
```diff
@@ -1,13 +1,15 @@
 package com.wikia.webdriver.testcases.facebooktests;
 
+import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.annotations.RelatedIssue;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.pages.login.SignInPage;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.AuthModal;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.signup.SignUpPageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialNewFilesPageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.login.SpecialUserLoginPageObject;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.special.login.SpecialUserLoginPageObject'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues:
* MAJOR 3 major
* MINOR 4 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/922853f6f8ad61cb19981b20e7be1325c2c17152/src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java) [rule](http://sonar-s1/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/922853f6f8ad61cb19981b20e7be1325c2c17152/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java) [rule](http://sonar-s1/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/922853f6f8ad61cb19981b20e7be1325c2c17152/src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java#L6) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* MAJOR 3 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/25aca1372c230b0b0cba583a09ff46d239bf07de/src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java) [rule](http://sonar-s1/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/25aca1372c230b0b0cba583a09ff46d239bf07de/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java) [rule](http://sonar-s1/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/25aca1372c230b0b0cba583a09ff46d239bf07de/src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java#L6) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java [hunk 5]
```diff
@@ -6,6 +6,7 @@
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.pages.login.RegisterPage;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.elements.mercury.pages.login.RegisterPage'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java [hunk 6]
```diff
@@ -57,27 +58,41 @@ public void anonCanLogInViaAuthModalWhenAddingVideo() {
...
-  @RelatedIssue(issueID = "SOC-2669", comment = "fails after product change, SOC team on it")
   public void anonCanLogInViaUserLoginPage() {
     WikiBasePageObject base = new WikiBasePageObject();
     base.openSpecialUpload(wikiURL);
     base.verifyLoginReguiredMessage();
     SpecialUserLoginPageObject special = base.clickLoginOnSpecialPage();
-    special.login(credentials.userName10, credentials.password10);
-    special.verifyUserLoggedIn(credentials.userName10);
+
+    SignInPage signInPage = new SignInPage(driver);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java]: MAJOR Remove this unused "signInPage" local variable. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java [hunk 7]
```diff
@@ -72,9 +77,16 @@ public void anonCanRemindPasswordOnUserLoginSpecialPageUsingLowerCaseUserName()
     String
         newPassword =
         login.receiveMailWithNewPassword(credentials.email, credentials.emailPassword);
-    signIn.login(userName, newPassword);
     String verifyString = userName.substring(0, 1).toUpperCase() + userName.substring(1);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java]: MAJOR Remove this unused "verifyString" local variable. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 2 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by justnpT** [comment]
Customized job with all the tests run:
http://jenkins:8080/view/Social_OLD/job/Test-Case-Automation-Customizable/

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditAccountTests.java [hunk 8]
```diff
@@ -29,15 +32,23 @@ public void EditAccount_001_closeAccount() {
...
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditAccountTests.java]: MAJOR Remove this unused "login" local variable. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by qaga** [comment]
LGTM

**Pull request merged**